### PR TITLE
Retry in case of all exception in start method

### PR DESCRIPTION
**Pull request by suranjan**
To avoid formatting changes, have put the try/catch block without formatting

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java [hunk 1]
```diff
@@ -72,6 +72,7 @@ public class PostgresConnectorTask extends BaseSourceTask<PostgresPartition, Pos
 
     @Override
     public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> start(Configuration config) {
+        try {
```
Comment (asrinivasanyb) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java:75]: indentation ?

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java [hunk 2]
```diff
@@ -268,14 +269,15 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
             coordinator.start(taskContext, this.queue, metadataProvider);
 
             return coordinator;
+        }
```
Comment (asrinivasanyb) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java:272]: Do not follow this change ? Just seems to have moved and formatting has changed ?

**Comment by asrinivasanyb** [comment]
With this change, which does seem to be required, can the changes from https://github.com/yugabyte/debezium/pull/121 be reverted as this will cover those changes as well

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java (hunk 2 above)
Comment (suranjan) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java:272]: inner try doesn't have catch but just finally. The catch block is for the overall try block.

**Pull request closed without merging**